apps/aecore/src/aec_peers_pool.erl

```diff
@@ -0,0 +1,1695 @@
+%%%=============================================================================
+%%% @copyright 2017, Aeternity Anstalt
+%%% @doc Stochastic pool.
+%%%
+%%% Opaque data structure that handle a collection of peers.
+%%% Implements the design described in:
+%%%   https://github.com/aeternity/protocol/blob/master/GOSSIP.md
+%%%
+%%% Usage:
+%%%  - When setting up, add trusted peers calling `update/7`.
```

Comment:
If we're only calling this upon startup, wouldn't `update_trusted/6` be more appropriate, less room for mistakes when using the API in the rest of the system.